AirCasting/SessionViews/SessionHeaderView.swift

```diff
@@ -66,8 +66,27 @@ private extension SessionHeaderView {
                     }
                 }
             }
-            Text("\(session.type?.description ?? SessionType.unknown("").description): \(session.deviceType?.description ?? "")")
-                .font(Font.moderate(size: 13, weight: .regular))
+            // As long as we get session.deviceType = nil we should handle somehow showing those devices which were used to record
+            // [|(-)   (-)|]    |-------------------|
+            //  |   ___   |  -- | You, do something |
+            //  |_________|     |-------------------|
+            // so the idea at leat for now is this below
```

Comment:
Monkey 💙
I'm voting to leave it and add an additional warning here (it'd be just a 43rd warning, no big deal xD).

Reply:
uh, i see the warning, good job ;D